common/defaultRetrySettings.go

```diff
@@ -0,0 +1,11 @@
+package common
+
+// DefaultRetrySettings indicates what the "default" retry settings
+// are if it is not specified on an Activity or for any unset fields
+// if a policy is explicitly set on a workflow
+type DefaultRetrySettings struct {
+	InitialIntervalInSeconds   int32
```

Comment:
We are not using `InSeconds` suffix for fields which represent Duration anymore.  Also use Duration as the type.